backend/typescript/yarn.lock

```diff
@@ -4,26 +4,26 @@
 
 "@babel/code-frame@7.12.11":
   version "7.12.11"
-  resolved "https://registry.yarnpkg.com/@babel/code-frame/-/code-frame-7.12.11.tgz#f4ad435aa263db935b8f10f2c552d23fb716a63f"
```

Comment:
Not sure why these changes are made in yarn.lock. 

Could someone verify if this is ok? @Tahmeed27 @Tayef-Shah

Reply:
Hmm its interesting that it changed to npm instead of yarn. Idk if that's because you did npm install at some point or not tho. You should only use yarn for dependencies and not mix it up with npm. Maybe you could delete yarn.lock and then run yarn again to see if that changes this file?